src/core/gltf2importer/gltf2parser.cpp

```diff
@@ -352,6 +379,10 @@ Qt3DCore::QEntity *GLTF2Parser::parse(const QByteArray &jsonData, const QString
         return nullptr;
     }
 
+    generateMissingTangents(basePath, rootObject);
```

Comment:
This step should still be controlled by a condition, I can image in some cases we might purposely not want to generate tangents

Reply:
we could make this a button for each mesh in the mesh collection?

Reply:
added a button in the MeshWidget for each mess that misses tangents